mysql-test/README

```diff
@@ -26,7 +41,8 @@ other relevant options.
 
 With no test cases named on the command line, mysql-test-run falls back
 to the normal "non-extern" behavior. The reason for this is that some
-tests cannot run with an external server.
+tests cannot run with an external server (because they need to control the
+options with which the server is started).
 
```

Comment:
I'd rather say
```
to the normal "non-extern" behavior. The reason for this is that many
tests cannot run with an external server (for example, because they need to
control the options with which the server is started). 
```
note s/some/many/ and "for example"

Reply:
ack. rebased and merged. seems part of this feedback was already done recently.